### DISCUSSION TITLE
How to show the slide zoomed by default

### DISCUSSION THREAD

**Discussion by mdinjamul** [igordanchenko/yet-another-react-lightbox#147] [Q&A] [answered]
He Community,
I have used this lightbox on my website, and I am happy with that, but I need a little help.
I want to show the slide images a little zoomed by default (transform: scale(5)), can anyone help me with how can I do that?

**Comment by igordanchenko** [comment]
Hi there! Can you describe the effect you are trying to achieve?

If you want to display your slides flush to the lightbox edges, you can change the `padding` prop:

```js
carousel={{ padding: 0 }}
```

But if you really want to scale your images with css, you can target the `yarl__slide_image` css class:

```css
.yarl__slide_image {
    transform: scale(1.05);
}
```

or 

```js
styles={{ slide: { transform: "scale(1.05)" } }}
```

**Reply by mdinjamul** [reply]
basically, the slide images displaying transform: scale(1) by default, I want to change it transform: scale(5) by default a little zoomed in by default. 
check the screenshot, you will get it.
Default slide width - https://prnt.sc/8Sezx6WjT7aO
I want this width - https://prnt.sc/lkJwS5db8cb9

**Reply by igordanchenko** [reply, marked as answer]
I'm afraid, this use case is not supported by Zoom plugin.
